products/caregivers/Improve Facility Selection/10-10 Health Apps-10-10CG-[date].md

```diff
@@ -2,71 +2,109 @@
 - Backend changes
-    + Does the project introduce any new or unusual infrastructure dependencies?
-    + Do you need to poll any APIs for status?
-    + Are you handling all failure and error cases while in custody of your users's data?
+  - Does the project introduce any new or unusual infrastructure dependencies?
+    - No
+  - Do you need to poll any APIs for status?
+    - No
+  - Are you handling all failure and error cases while in custody of your users's data?
+    - Yes
 - Internal API changes
-    + List new or modified APIs in `vets-api`
-    + Are you deprecating or removing any APIs?
-    + Do you have API documentation?
-    + Describe expected call patterns
+  - List new or modified APIs in `vets-api`
+    - `/v0/caregivers_assistance_claims/facilities` uses the Lighthouse facilities api endpoint
+  - Are you deprecating or removing any APIs?
+    - No
+  - Do you have API documentation?
+    - No, the only change is a new endpoint in the caregivers controller.
+  - Describe expected call patterns
+    - `vets-website` calls this endpoint to a list of paginated facilities
 - External API changes
-    + List new or modified APIs for upstream or external systems
-    + Describe expected call patterns
-    + What PII or PHI will be transmitted to/from the external systems?
+  - List new or modified APIs for upstream or external systems
+    - No Changes
+  - Describe expected call patterns
+    - No Changes
+  - What PII or PHI will be transmitted to/from the external systems?
+    - No Changes
+  - List any required background processing
+    - No Changes
+  - Describe error and dead letter handling
+    - No Changes
 - Background jobs
-    + List any required background processing
-    + Describe error and dead letter handling
+  - List any required background processing
+  - Describe error and dead letter handling
 - Data storage
-    + Describe new or modified databases, tables or columns
-    + Describe indexes and constraints
-    + Identify PII and PHI and where and how it will be stored, processed, expired and deleted
+  - Describe new or modified databases, tables or columns
+    - No Changes. The facility that is ultimately returned is used to pass the facility id as part of the 1010CG Form submission.
+  - Describe indexes and constraints
+    - No Changes
+  - Identify PII and PHI and where and how it will be stored, processed, expired and deleted
+    - No Changes
 - Libraries and dependencies
-    + List new or updated dependencies
+  - List new or updated dependencies
+    - Lighthouse Facilities API (New to the 10-10CG, but used in VA.gov)
+    - Mapbox API (New to the 10-10CG, but used in VA.gov)
 - Metrics, logging, observability, alerting
-    + Identify key areas to monitor
+  - Identify key areas to monitor
+    - Monitor api status for mapbox api and Lighthouse facilities api
 - Infrastructure and network changes
-    + List any changes or additions
```

Comment:
Do we have any changes or additions to Infra or network?  it doesnt really sound like it to me, but i am certainly no reliable source

Reply:
I don't think so? We don't use any custom credentials or anything (we use the same ones that vets-api use for the two apis we are hitting); that would be the only infra thing I could think of.